recipes/xerces-c/all/patches/0001-usercurl-cmake-targets.patch

```diff
@@ -0,0 +1,24 @@
+From af1935b567c18c8b2de71d6ea97239be60876335 Mon Sep 17 00:00:00 2001
+From: Chris Mc <prince.chrismc@gmail.com>
+Date: Fri, 3 Sep 2021 20:21:37 -0400
+Subject: [PATCH] link to installed CMake targets of CURL
+https://github.com/apache/xerces-c/pull/34
+
+Just like how it was done for ICU
+---
+ src/CMakeLists.txt | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index 344851fad..f396fc5ef 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -1085,7 +1085,7 @@ endif()
+ if(XERCES_USE_NETACCESSOR_CURL)
+   list(APPEND libxerces_c_SOURCES ${curl_sources})
+   list(APPEND libxerces_c_HEADERS ${curl_headers})
+-  list(APPEND libxerces_c_DEPS ${CURL_LIBRARIES})
++  list(APPEND libxerces_c_DEPS CURL::libcurl)
+ endif()
+ 
+ if(XERCES_USE_NETACCESSOR_SOCKET)
```

Comment:
this patch is useless, use `cmake_find_package` instead of `cmake_find_package_multi`

Reply:
`${CURL_LIBRARIES}` was missing the OpenSSL, I dont see the difference the generator would make 🤔

Reply:
`CURL_LIBRARIES` doesn't miss OpenSSL if you use cmake_find_package. Also I'm pretty sure that files generated by cmake_find_package_multi are completly ignored by upstream CMakeLists, because FindICU.cmake and FindCURL.cmake are provided in CMake installation and have precedence over config files.

Reply:
@SpaceIm you're right, this patch is not strictly necessary. Using the CMake provided FindCURL.cmake is enough, as you said. See my latest two commits ( ee913ba, 3ad9499)

Reply:
Please use `FindCURL.cmake` provided by conan, it's more robust, so replace `cmake_find_package_multi` by `cmake_find_package`.

Reply:
It was a required step of the development processes 😆 it's much cleaner :+1:

You can also delete the `FindICU` since it should never be used with something like
`tools.remove_files_by_mask(os.path.join(self._sources_subfolder, "cmake"), "Find*.cmake")`